applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -106,7 +121,12 @@ extern "C" PyObject * BaseContext_createObject_Impl(PyObject * self, PyObject *
             {
                 if (PyString_Check(value))
                     desc.setAttribute(PyString_AsString(key),PyString_AsString(value));
-                else
+                else if (PyList_Check(value)){
```

Comment:
About the sequence type... @maxime-tournier  is right this is the way to go.

About the SofaPythonListToStdStr, we probably need to implement a more "precise" conversion. 
Currently [1, 2 ,3, [4, 5]] will be converted to "1 2 3 [4, 5]"  which is not exactly what we want and will not report useful warning/error message to the user.   

So, I suggest you to add more check for each list entry so they can be converted and if the sequence content cannot be converted it would be better to report at this layer a good looking error message (with the faulty python filename/line of code).